client/src/components/all-components/InformationLetterEmail.js

```diff
@@ -88,14 +91,21 @@ const InformationLetterEmail = (props) => {
               </div>
               <div className="tw-w-full tw-mx-auto tw-bg-[#B4B4B4] tw-h-[2px] tw-my-4 tw-opacity-40" />
```

Comment:
possible to add a little drop shadow to make the letter stand out from the white background a little better?